src/dds/rs-dds-auto-calibration.cpp

```diff
@@ -0,0 +1,97 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2024 Intel Corporation. All Rights Reserved.
+
+
+#include "rs-dds-auto-calibration.h"
+#include <rsutils/string/from.h>
+#include <rsutils/json.h>
+#include <src/ds/d500/d500-auto-calibration.h>
+
+
+namespace librealsense
+{
+dds_auto_calibrated::dds_auto_calibrated() :
+    _auto_calib_capability(std::make_shared<d500_auto_calibrated>()) {}
```

Comment:
Not all DDS devices will be D555 - in that case, TC should be disabled.
Why not pass it in, so the actual class that derives from us can decide on an implementation, if any?

Reply:
done